src/main/java/com/googleinterns/zoomtube/data/Comment.java

```diff
@@ -24,34 +24,11 @@
 /** Contains data related to a comment in a discussion. */
 @AutoValue
 public abstract class Comment {
-  public static final String ENTITY_KIND = "Comment";
-  public static final String PROP_LECTURE = "lecture";
-  public static final String PROP_PARENT = "parent";
-  public static final String PROP_TIMESTAMP = "timestamp";
-  public static final String PROP_AUTHOR = "author";
-  public static final String PROP_CONTENT = "content";
-  public static final String PROP_CREATED = "created";
-
-  private static Comment create(Key key, Key lecture, Optional<Key> parent, Date timestamp,
+  public static Comment create(Key key, Key lecture, Optional<Key> parent, Date timestamp,
```

Comment:
Do you think you can rename `key` and `lecture` to be `commentKey` and `lectureKey` just so there won't be any confusion.

Reply:
Done!